src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentiveMappingService.kt

```diff
@@ -28,11 +29,9 @@ class IncentiveMappingService(@Qualifier("mappingApiWebClient") private val webC
       .bodyToMono(IncentiveNomisMapping::class.java)
       .onErrorResume(WebClientResponseException.NotFound::class.java) {
         Mono.empty()
-      }
-      .block()
```

Comment:
apologies I went a bit overboard with these changes.  I've tried to remove all the blocking and instead `suspend`.  The listeners now return a `CompletableFuture` so that they are async too